client-lib/go/client/jvs_client.go

```diff
@@ -57,5 +65,42 @@ func NewJVSClient(ctx context.Context, config *JVSConfig) (*JVSClient, error) {
 
 // ValidateJWT takes a jwt string, converts it to a JWT, and validates the signature.
 func (j *JVSClient) ValidateJWT(jwtStr string) (*jwt.Token, error) {
+	// Handle unsigned tokens.
+	if strings.HasSuffix(jwtStr, UnsignedPostfix) {
+		token, err := jwt.Parse([]byte(jwtStr), jwt.WithVerify(false))
+		if err != nil {
+			return nil, fmt.Errorf("failed to parse jwt %s: %w", jwtStr, err)
+		}
+		valid, err := j.unsignedTokenValidAndAllowed(token)
```

Comment:
From this usage, it doesn't seem the `bool` return is necessary? Having an err is equivalent to having a false.